live-examples/css-examples/basic-box-model/clear.css

```diff
@@ -0,0 +1,21 @@
+.example-container {
+    background-color: #eee;
+    border: .75em solid;
+    padding: .75em;
+    text-align: left;
+}
+
+.floated-left {
+    border: solid 10px #FFC129;
+    background-color: rgba(255, 244, 219, 0.6);
```

Comment:
We should omit leading zeroes, here and elsewhere in this PR.

Reply:
fixed.